targets/DEV1-TMU/main.cpp

```diff
@@ -95,7 +96,34 @@ int main() {
 
     // Initialize the timer
     DEV::Timer& timer = DEV::getTimer<DEV::MCUTimer::Timer2>(100);
-    TMU::TMU tmu;
+
+    /**
+     * Initialize SPI
+     */
+    const uint8_t deviceCount = 4;
+    IO::GPIO* devices[deviceCount];
+
+    devices[0] = &IO::getGPIO<IO::Pin::PB_4>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    devices[1] = &IO::getGPIO<IO::Pin::PB_5>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    devices[2] = &IO::getGPIO<IO::Pin::PB_6>(EVT::core::IO::GPIO::Direction::OUTPUT);
+    devices[3] = &IO::getGPIO<IO::Pin::PB_7>(EVT::core::IO::GPIO::Direction::OUTPUT);
+
+    devices[0]->writePin(IO::GPIO::State::HIGH);
+    devices[1]->writePin(IO::GPIO::State::HIGH);
+    devices[2]->writePin(IO::GPIO::State::HIGH);
+    devices[3]->writePin(IO::GPIO::State::HIGH);
+
+    IO::SPI& spi = IO::getSPI<IO::Pin::PA_5, IO::Pin::PA_7, IO::Pin::PA_6>(devices, deviceCount);
+
+    spi.configureSPI(SPI_SPEED_1MHZ, SPI_MODE0, SPI_MSB_FIRST);
+
+    TMU::DEV::MAX31855 therms[4] = {
+        TMU::DEV::MAX31855(spi, 0),
+        TMU::DEV::MAX31855(spi, 1),
+        TMU::DEV::MAX31855(spi, 2),
+        TMU::DEV::MAX31855(spi, 3)};
```

Comment:
```suggestion
        TMU::DEV::MAX31855(spi, 3),
    };
```
This will just make the autoformatting a bit nicer